pkg/thing/mocks/connector.go

```diff
@@ -0,0 +1,27 @@
+package mocks
+
+import (
+	"github.com/CESARBR/knot-babeltower/pkg/thing/entities"
+	"github.com/stretchr/testify/mock"
+)
+
+type FakeConnector struct {
+	mock.Mock
+	SendError error
+	RecvError error
+}
+
+func (fc *FakeConnector) SendRegisterDevice(id, name string) (err error) {
+	ret := fc.Called(id, name)
+	return ret.Error(0)
+}
+
+func (fc *FakeConnector) SendUpdateSchema(id string, schemaList []entities.Schema) (err error) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [exported method FakeConnector.SendUpdateSchema should have comment or be unexported](https://app.codacy.com/manual/joaoaneto/knot-babeltower/pullRequest?prid=4853780)